ludwig/schema/features/preprocessing/binary.py

```diff
@@ -17,7 +26,7 @@ class BinaryPreprocessingConfig(BasePreprocessingConfig):
     """BinaryPreprocessingConfig is a dataclass that configures the parameters used for a binary input feature."""
 
     missing_value_strategy: str = schema_utils.StringOptions(
-        MISSING_VALUE_STRATEGY_OPTIONS + [FILL_WITH_FALSE],
+        [FILL_WITH_MODE, BFILL, FFILL, DROP_ROW, FILL_WITH_FALSE, FILL_WITH_TRUE],
```

Comment:
Note: This will break older models that have `missing_value_strategy: FILL_WITH_CONST`. However, 1) the risk seems low since this wasn't the default value and 2) these models would be broken anyway unless they had also specified `fill_value`.